src/renderer/themes.css

```diff
@@ -237,6 +242,25 @@ it can be safely elided. This looks quite pleasant on this theme. */
   text-decoration: underline;
 }
 
+.nordic {
+  --primary-text-color: #EEEEEE;
+  --secondary-text-color: #ddd;
+  --tertiary-text-color: #EEEEEE;
+  --title-color: #EEEEEE;
+  --bg-color: #2b2f3a;
+  --favorite-icon-color: #85a4c1;
```

Comment:
```suggestion
  --favorite-icon-color: #FFEA00;
```

Reply:
@JonahFoster Here's my suggestion (same as what we have for dark mode). I think we as a whole should  change our setup to one favorite color for light themes and one favorite color for darker themes, but that's certainly out of scope for this issue. Thank you!